providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
```

Comment:
Is it worth setting up Dependabot in these module repos, to make sure we don't trail behind?

Reply:
This might be a good idea! With the `~> 5.0` syntax, we will always default to the latest minor release, so the Dependabot would only have to keep track of the major AWS provider releases which don't happen very often but it could still be beneficial.